contracts/modules/STO/USDTiered/USDTieredSTOStorage.sol

```diff
@@ -6,7 +6,9 @@ pragma solidity ^0.5.0;
 contract USDTieredSTOStorage {
 
     bytes32 internal constant INVESTORSKEY = 0xdf3a8dd24acdd05addfc6aeffef7574d2de3f844535ec91e8e0f3e45dba96731; //keccak256(abi.encodePacked("INVESTORS"))
-
+    bytes32 constant ADMIN = "ADMIN";
+    bytes32 constant OPERATOR = "OPERATOR";
```

Comment:
please verify if this is a duplicate declaration. `OPERATOR` and `ADMIN` are already defined in `ModuleStorage` inherited via `STO -> Module -> ModuleStorage`